specs/ov_management_api.md

```diff
@@ -0,0 +1,171 @@
+# Ownership Voucher Management API
+
+**STATUS: Draft**
+
+This specification describes a protocol that (authenticated) external parties can use to manage the Ownership Vouchers (OV) and Owner Onboarding Server tracks.
+This protocol will be called the "Ownership Voucher Management API", or in the scope of this document, the "Management API".
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+RFC 2119.
+
+## General Features
+
+### Encoding
+
+Requests and responses for the Management API use JSON encoding, instead of the CBOR encoding used throughout the FIDO specification.
+
+### Authentication
+
+Multiple authentication methods are specified in this document, and a server can support any combination of them.
+Clients will need to interact with Management API server operators to determine which authentication method to use, and to obtain credentials.
+The server will log who performs actions, and which authentication method was used, but it is not required to store this information in its data stores.
+The method by which clients are provided with credentials is outside the scope of this specification.
+
+#### X.509 Client Certificate
+
+In this authentication method, a client will send its certificate as part of the TLS connection handshake.
+For this, the certificate needs to be signed by a Certificate Authority that the server trusts.
+The `username` in the protocol is the value of the `CN` field in the `Subject` of the certificate.
+
+#### OAuth2 Bearer token in Authorization header
+
+In this authentication method, the client will send an OAuth2 Bearer token, as per [RFC6750, section 2.1](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1).
+How the server determines a `username` for this authentication method is outside the scope of this specification.
+
+### Responses
+
+Requests that got parsed and executed succesfully will have a Status code in the Succesful range (section 6.3, RFC 7231).
```

Comment:
Small thing but as in the previous section we've included a link to the RFC, I find it consistent (and useful!) to include them here as well:

- https://datatracker.ietf.org/doc/html/rfc7231#section-6.3
- https://datatracker.ietf.org/doc/html/rfc7231#section-6.5
- https://datatracker.ietf.org/doc/html/rfc7231#section-6.6